test-support/helpers/ember-basic-dropdown.js

```diff
@@ -54,7 +54,7 @@ export function clickTrigger(scope, options = {}) {
 }
 
 export function tapTrigger(scope, options = {}) {
-  let selector = '.ember-basic-dropdown-trigger';
+  let selector = `.ember-basic-dropdown-trigger ${options.triggerChildSelector || ''}`.trim();
```

Comment:
Instead of defining this ad-hoc option, you can use the `nativeTap` function I've just added to trigger a tap in the element inside.